src/api/responses.rs

```diff
@@ -0,0 +1,254 @@
+        count: usize,
+        #[serde(rename = "childrenMessageIds")]
+        children_message_ids: Vec<Record>,
+    },
+    /// Response of GET /api/<keyspace>/messages?<index>
+    MessagesForIndex {
+        index: String,
+        #[serde(rename = "maxResults")]
+        max_results: usize,
+        count: usize,
+        #[serde(rename = "messageIds")]
+        message_ids: Vec<String>,
+    },
+    /// Response of GET /api/<keyspace>/messages?<index>[&expanded=true]
+    MessagesForIndexExpanded {
+        index: String,
+        #[serde(rename = "maxResults")]
+        max_results: usize,
+        count: usize,
+        #[serde(rename = "messageIds")]
+        message_ids: Vec<Record>,
+    },
+    /// Response of GET /api/<keyspace>/messages?<index>
+    MessagesForTag {
+        tag: String,
+        #[serde(rename = "maxResults")]
+        max_results: usize,
+        count: usize,
+        #[serde(rename = "messageIds")]
+        message_ids: Vec<String>,
+    },
+    /// Response of GET /api/<keyspace>/messages?<index>[&expanded=true]
+    MessagesForTagExpanded {
+        tag: String,
+        #[serde(rename = "maxResults")]
+        max_results: usize,
+        count: usize,
+        #[serde(rename = "messageIds")]
+        message_ids: Vec<Record>,
+    },
+    /// Response of GET /api/<keyspace>/addresses/<address>/outputs
+    OutputsForAddress {
+        address: String,
+        #[serde(rename = "maxResults")]
+        max_results: usize,
+        count: usize,
+        #[serde(rename = "outputIds")]
+        output_ids: Vec<String>,
+    },
+    /// Response of GET /api/<keyspace>/addresses/<address>/outputs[?expanded=true]
+    OutputsForAddressExpanded {
+        address: String,
+        #[serde(rename = "maxResults")]
+        max_results: usize,
+        count: usize,
+        #[serde(rename = "outputIds")]
+        output_ids: Vec<Record>,
+    },
+    /// Response of GET /api/<keyspace>/outputs/<output_id>
+    Output {
+        #[serde(rename = "messageId")]
+        message_id: String,
+        #[serde(rename = "transactionId")]
+        transaction_id: String,
+        #[serde(rename = "outputIndex")]
+        output_index: u16,
+        #[serde(rename = "spendingTransaction")]
+        spending_transaction: Option<Value>,
+        output: Value,
+    },
+    /// Response of GET /api/<keyspace>/transactions/<message_id>
+    Transaction(Transaction),
+    /// Response of GET /api/<keyspace>/transactions/ed25519/<address>
+    Transactions { transactions: Vec<Transaction> },
+    TransactionHistory {
+        address: String,
+        transactions: Vec<Transfer>,
+    },
+    /// Response of GET /api/<keyspace>/milestone/<index>
+    Milestone {
+        #[serde(rename = "index")]
+        milestone_index: u32,
+        #[serde(rename = "messageId")]
+        message_id: String,
+        timestamp: u64,
+    },
+    /// Response of GET /analytics/addresses[?start_timestamp=<u32>&end_timestamp=<u32>]
```

Comment:
Maybe this endpoint should also be consistent with other api calls? Like
`/api/<api_version>/analytics/addresses[?start_timestamp=<u32>&end_timestamp=<u32>]`.